protocol/packages/marketprice/Cargo.toml

```diff
@@ -6,9 +6,12 @@ edition.workspace = true
 license.workspace = true
 
 [features]
+astroport = ["dep:currencies", "currencies/astroport"]
+osmosis = ["dep:currencies", "currencies/osmosis"]
```

Comment:
`"dep:currencies"` looks unnecessary

Reply:
I added it just to hide `currencies` from the features list. I can remove it if you don't mind it being exposed there.

Reply:
probably do not get you right 
> I added it just to hide currencies from the features list
I believe it's not necessary to have a separate feature. It's mandatory to have `currencies` in the `[dependencies]` though.

Reply:
If an optional dependency is not mentioned by a feature, then a new feature with the name of the dependency is automatically created. By doing this, I am trying to avoid creating that feature as it is not functional by any means. I haven't tested whether I can only go with `currencies/protocol`, so I will try it out and report back.

Reply:
I am pretty sure we can go on;y with `currencies/protocol`

Reply:
No, unless there is `dep:` it creates it. What should I do in this case?

Reply:
![image](https://github.com/nolus-protocol/nolus-money-market/assets/21364974/633419f9-378f-4f63-bf5e-700c0aa577db)

Reply:
so no sense of specifying `dep:currencies` in our case

Reply:
> No, unless there is `dep:` it creates it. What should I do in this case?

just leave `osmosis = [currencies/osmosis]` and the corresponding one for astroport